projects/testing-library/tests/issues/issue-280.spec.ts

```diff
@@ -0,0 +1,55 @@
+import {Component, NgModule} from '@angular/core';
+import {render, screen} from '../../src/public_api';
+import {RouterModule, Routes} from "@angular/router";
+import {RouterTestingModule} from "@angular/router/testing";
+import {click} from "@testing-library/user-event/dist/click";
+import {Location} from "@angular/common";
+
+@Component({
+  template: `<div>Navigate</div> <router-outlet></router-outlet>`,
+})
+class MainComponent {}
+
+@Component({
+  template: `<div>first page</div><a routerLink="/second">go to second</a>`
+})
+class FirstComponent {}
+
+@Component({
+  template: `<div>second page</div><button (click)="goBack()">navigate back</button>`
+})
+class SecondComponent {
+  constructor(private location: Location) { }
+  goBack() {this.location.back();}
+}
+
+const routes: Routes = [
+  {path: '', redirectTo: '/first', pathMatch: 'full'},
+  {path: 'first', component: FirstComponent},
+  {path: 'second', component: SecondComponent}
+];
+
+@NgModule({
+  declarations: [FirstComponent, SecondComponent],
+  imports: [RouterModule.forRoot(routes)],
+  exports: [RouterModule]
+})
+class AppRoutingModule {}
+
+
+test('navigate to second page and back', async () => {
+  const subject = await render(MainComponent, {imports: [AppRoutingModule, RouterTestingModule]});
+  await subject.navigate('/');
+
+  expect(await screen.findByText('Navigate')).toBeTruthy();
```

Comment:
I left extra expects in place, showing that there is successful navigation to second page.
I typically don't do that in my tests, please let me know if you want me to remove extras.

Thank you.

Reply:
This is fine, thanks!